### ISSUE TITLE
ONNXImporter ERROR while importing custom mobilenet_ssd model with 

### ISSUE THREAD

**Issue by hlreicha** [opencv/opencv#20634]
##### System information (version)
- OpenCV => 4.5.3
- Operating System/Platform: Ubuntu 18.04/Python 3.9

##### Detailed description
I followed these [instructions](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html) to train my own custom object detector using a pre-trained model. My pretrained model is ssdlite_mobilenet_v2_coco_2018_05_09 and I grabbed this model from [here](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md). I ran the export_inference_graph.py script to convert my checkpoint files to a save_model.pb. I would like to use opencv dnn module to inference on my model. I downloaded the tensorflow-onnx library and ran this command:
```
       python -m tf2onnx.convert --saved-model <path_to_model>/save_model --opset 11 --output model.onnx
```
I written a simple program to just load the onnx file to opencv
```
import cv2

net=cv2.dnn.readNetFromONNX('./model.onnx')
```
I receive this error when I run my program
```
cv2.error: OpenCV(4.5.3) /tmp/pip-req-build-3umofm98/opencv/modules/dnn/src/onnx/onnx_importer.cpp:2146: error: (-2:Unspecified error) in function 'handleNode'
> Node [Conv]:(StatefulPartitionedCall/ssd_mobile_net_v2fpn_keras_feature_extractor/functional_1/bn_Conv1/FusedBatchNormV3:0) parse error: OpenCV(4.5.3) /tmp/pip-req-build-3umofm98/opencv/modules/dnn/src/layers/convolution_layer.cpp:96: error: (-213:The function/feature is not implemented) Unsupported asymmetric padding in convolution layer in function 'BaseConvolutionLayerImpl'
```
I only ever managed to inference through Tensorflow on the checkpoint files and they worked as they should. I would appreciate any help or feedback if possible. The model and the program is [here](https://drive.google.com/file/d/1RqXWd0tQ_Whuv_DXiqiRCmTWVuGIzjIY/view?usp=sharing)

##### Issue submission checklist

 - [x] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues,
       forum.opencv.org, Stack Overflow, etc and have not found solution
   
 - [x] I updated to latest OpenCV version and the issue is still there
   
 - [x] There is reproducer code and related data files: videos, images, onnx, etc

**Comment by SamFC10** [comment]
> Unsupported asymmetric padding in convolution layer

Has been fixed by #20394. Please try building opencv from source with latest master branch.

**Comment by sainivedh19pt** [comment]
Hi @SamFC10,

Facing the same issue while loading yolox - onnx file, have you found a fix for the same. 

Thanks

**Comment by asmorkalov** [comment]
It looks like the original issue has been fixed. OpenCV does not support dynamic shapes for now. Please fix input shape and report the issue status.

**Comment by sainivedh19pt** [comment]
Hi @asmorkalov, The issue is resolved now.

Thanks

**Comment by caobin2019-cmd** [comment]
> > Unsupported asymmetric padding in convolution layer
> 
> Has been fixed by #20394. Please try building opencv from source with latest master branch. Different error when I try to run your model :
> 
> ```
> Node [Unsqueeze]:(Unsqueeze__1020:0) parse error: OpenCV(4.5.3-dev) /home/jebastin/opencv_build/opencv/modules/dnn/src/onnx/onnx_importer.cpp:1822: error: (-213:The function/feature is not implemented) Multidimensional unsqueeze in function 'parseUnsqueeze'
> ```

so how to save the question you reported about the "Multidimensional unsqueeze in function 'parseUnsqueeze"
